legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/planner/BitemporalDeltaPlanner.java

```diff
@@ -93,10 +97,12 @@ class BitemporalDeltaPlanner extends BitemporalPlanner
     private List<FieldValue> primaryKeyFieldsAndFromField;
     private List<FieldValue> dataFields;
 
-    private final String startAlias = "start_date";
-    private final String endAlias = "end_date";
-    private final String xAlias = "x";
-    private final String yAlias = "y";
+    private static final String START_DATE = "start_date";
+    private static final String END_DATE = "end_date";
+    private static final String X_ALIAS = "x";
+    private static final String Y_ALIAS = "y";
```

Comment:
Given we are defining this as constant, can we provide more descriptive names.  Ideally, the purpose/intention of these could be encapsulated on the name, helping self-documenting the code.